android/src/main/java/com/reactcommunity/rnlocalize/RNLocalizeModule.java

```diff
@@ -197,6 +198,26 @@ private WritableMap getNumberFormatSettings(Locale locale) {
     return settings;
   }
 
+  private boolean isAutoDateAndTime() {
```

Comment:
I find

```java
private boolean isAutoTimeZone() {
    ContentResolver resolver = getReactApplicationContext().getContentResolver();

    return (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1
        ? Settings.Global.getInt(resolver, Settings.Global.AUTO_TIME, 0)
        : Settings.System.getInt(resolver, Settings.System.AUTO_TIME, 0)) != 0;
}
```

more elegant, but it's just because I am used to immutability 😄

Reply:
sure!

Reply:
almost forgot this one - just added it to the PR feedback commit and re-pushed